etrago/appl.py

```diff
@@ -76,7 +75,10 @@
     'network_clustering_kmeans': {
         'active': True, # choose if clustering is activated
         'n_clusters': 30, # number of resulting nodes
-        'n_clusters_gas': 30, # number of resulting nodes
+        'cluster_foreign_gas': True, # take foreign gas buses into account, True or False
```

Comment:
`# take foreign gas buses into account` sounds a bit misleading, it could also mean that foreign gas buses are not considered at all. Maybe sth. like `# cluster foreign gas buses` would be better.

Reply:
Done